redfish-core/lib/fabric_adapters.hpp

```diff
@@ -225,6 +226,7 @@ inline void doAdapterGet(const std::shared_ptr<bmcweb::AsyncResp>& aResp,
     getFabricAdapterAsset(aResp, serviceName, fabricAdapterPath);
     getFabricAdapterState(aResp, serviceName, fabricAdapterPath);
     getFabricAdapterHealth(aResp, serviceName, fabricAdapterPath);
+    getLocationIndicatorActive(aResp, fabricAdapterPath);
```

Comment:
Can you debug more why we are getting this property even when it is null?

Reply:
Yes, I'm still investigating that issue.